generators/server/templates/_pom.xml

```diff
@@ -453,7 +453,7 @@
             <artifactId>hibernate-envers</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.hibernate</groupId>
+            <groupId>org.hibernate.validator</groupId>
```

Comment:
use new validator lib to validate new constrains as `javax.validation.constraints.Email` and `javax.validation.constraints.NotNull`